src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/seed/cas1/ApprovedPremisesRoomsSeedFromXLSXJob.kt

```diff
@@ -26,11 +26,15 @@ class ApprovedPremisesRoomsSeedFromXLSXJob(
   private val approvedPremisesRepository: ApprovedPremisesRepository,
   private val roomRepository: RoomRepository,
   private val bedRepository: BedRepository,
-  private val siteSurvey: SiteSurvey,
+  private val characteristicRepository: CharacteristicRepository,
 ) : ExcelSeedJob {
   private val log = LoggerFactory.getLogger(this::class.java)
+  private val questionCriteriaMapping = QuestionCriteriaMapping(characteristicRepository)
```

Comment:
if we want this to be newly constructed on every run of the seed job this will need to be constructed when the seed job is executed e.g. in processDataFrame, and then 'passed around' to functions that need it